src/server_manager/cloud/digitalocean_api.ts

```diff
@@ -139,6 +140,9 @@ export class RestApiSession implements DigitalOceanSession {
           user_data: dropletSpec.installCommand,
           tags: dropletSpec.tags,
           ipv6: true,
+          // We install metrics and droplet agents in the user_data script in order to not delay the droplet readiness.
```

Comment:
I'm curious what will the user observe? A improvement of the server creation time?

Reply:
The `monitoring: false` is a no-op, as that's the default value.
`with_droplet_agent: false` reduces the creation time, as the default installs that agent.